sormas-api/src/main/resources/captions_de-CH.properties

```diff
@@ -58,7 +58,8 @@ remove=Entfernen
 reportingUser=Meldender Nutzer
 
 # About
-aboutSormasVersion=SORMAS-Version
+aboutSormasVersion=Version
```

Comment:
please revert the changed to the translation files
they will be added by crowdin, you only need to modify the captions.properties